askama_actix/src/lib.rs

```diff
@@ -9,7 +9,7 @@ use actix_web::body::BoxBody;
 use actix_web::http::header::HeaderValue;
 use actix_web::http::StatusCode;
 use actix_web::{HttpResponse, HttpResponseBuilder, ResponseError};
-pub use askama::*;
+pub use askama::{self, *};
```

Comment:
Why do we need both? Can we just keep it as `*` and adjust the macro paths accordingly?

(Also I'd like this third commit to be squashed into the first one.)

Reply:
Well, that's overlooked. Fixed in the following (preview) commit (they will be squashed once review is done).